services/Sport.js

```diff
@@ -0,0 +1,29 @@
+const SportEntity = require('../entities/Sport');
+const Sport = require('../models/sports');
+
+const getAllSports = async() => {
+    const sportsData = await SportEntity.findAllSports();
+    sportsNames = sportsData.map(sport => sport.label);
+    return sportsNames;
```

Comment:
```suggestion
    return sportsData.map(sport => sport.label);
```
sportsNames was not even declared